lib/src/rule_component.html

```diff
@@ -0,0 +1,14 @@
+<div class="mb-3" *ngIf="view == View.listView">
+    <div class="flex-column">
+        <rule-list (detailedRule)="changeDetailedRule($event)" class="mb-5"></rule-list>
+        <button type="button" (click)="toCreateView()" class="mt-3 btn btn-success">Create Rule</button>
+    </div>
+</div>
+<div *ngIf="view == View.createView">
+    <rule-creator></rule-creator>
+    <button type="button" (click)="toListView()" class="btn btn-secondary">Back</button>
+</div>
```

Comment:
does Git have a delta diff view which omits the diff from previous RP?

Reply:
I don't think so :/